tests/CMakeLists.txt

```diff
@@ -256,7 +256,7 @@ file(GLOB_RECURSE COOKBOOK_TEST_SOURCES "${CMAKE_SOURCE_DIR}/cookbooks/*/*.wb")
 foreach(test_source ${COOKBOOK_TEST_SOURCES})
   get_filename_component(test_name ${test_source} NAME_WE)
   get_filename_component(test_dir ${test_source} DIRECTORY)
-  set(TEST_ARGUMENTS "${test_dir}/${test_name}.wb\;${test_dir}/${test_name}.grid")
+  set(TEST_ARGUMENTS "${test_dir}/${test_name}.wb\;${test_dir}/${test_name}.grid\;--resolution-limit\;75")
```

Comment:
20?

Reply:
I am not sure 20 will be enough to ensure hitting any not huge feature in a model. 50?

Reply:
deal.